### DISCUSSION TITLE
Disable/Redirect IP based access

### DISCUSSION THREAD

**Discussion by ajmnz** [dokku/dokku#4984] [Q&A]
I currently have two apps under `staging.domain.tld` and `api.staging.domain.tld`, but if instead of using those I use the server's IP address, I'm presented with the default Nginx page. `sites-enabled/default` is already unlinked.

Is there any way to disable that (return 404 or whatever) or redirect to any of the apps' domains? I guess I can directly edit the main Nginx config, but maybe there's a way to do it _within Dokku_.

Thanks!

**Comment by illgitthat** [comment]
I know this question is very old, but in case anyone else comes across this there is documentation on suggested methods here: https://dokku.com/docs/networking/proxies/nginx/
